alacritty/src/event.rs

```diff
@@ -1426,46 +1426,33 @@ impl Processor {
         }
     }
 
-    /// Create initial window and load GL platform.
-    ///
-    /// This will initialize the OpenGL Api and pick a config that
-    /// will be used for the rest of the windows.
-    pub fn create_initial_window(
-        &mut self,
-        event_loop: &EventLoopWindowTarget<Event>,
-        proxy: EventLoopProxy<Event>,
-        options: WindowOptions,
-    ) -> Result<(), Box<dyn Error>> {
-        let window_context = WindowContext::initial(
-            event_loop,
-            proxy,
-            self.config.clone(),
-            options,
-            #[cfg(all(feature = "wayland", not(any(target_os = "macos", windows))))]
-            self.wayland_event_queue.as_ref(),
-        )?;
-
-        self.windows.insert(window_context.id(), window_context);
-
-        Ok(())
-    }
-
     /// Create a new terminal window.
     pub fn create_window(
         &mut self,
         event_loop: &EventLoopWindowTarget<Event>,
         proxy: EventLoopProxy<Event>,
         options: WindowOptions,
     ) -> Result<(), Box<dyn Error>> {
-        let window = self.windows.iter().next().as_ref().unwrap().1;
-        let window_context = window.additional(
-            event_loop,
-            proxy,
-            self.config.clone(),
-            options,
-            #[cfg(all(feature = "wayland", not(any(target_os = "macos", windows))))]
-            self.wayland_event_queue.as_ref(),
-        )?;
+        let window_context = if let Some(window) = self.windows.iter().next().as_ref() {
+            let window = window.1;
+            window.additional(
+                event_loop,
```

Comment:
This is not entirely correct though, because you'll try to reload the so built display that way, which is not what we want, because we don't terminate the OpenGL. What we want is to keep the display/config from the initial window somewhere around and don't do all the recreations.